README.md

```diff
@@ -1 +1,32 @@
-# burger_town
+# BurgerTown
+
+> BurgerTown is a User Interface created with ReactApp that utilizes the [Bob's Burgers API](https://www.bobsburgersapi.com/) to access all things Bob's Burgers.
```

Comment:
ReactApp isn't a thing, to my knowledge. There's React, which is the library itself, and Create React App, which is the service that generates React projects. I think you want the latter here